fastlane/generateDocs.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+source ./replace.sh
```

Comment:
Why only separate a single function into another file? Variables can be declared locally inside it.
```suggestion
substitute() {
    match=""
    replace=match
    file=""
    appedTo=""

    while [ $# -gt 0 ]
    do
        if [[ $1 == **"--"** ]]
        then
            param="${1/--/}"
            declare $param="$2"
        fi
        shift
    done

    if [[ "$appendTo" == "" ]]
    then
        sed -i -E "s/$match/$replace/g" $file
    else [[ "$appendTo" != "" ]]
        sed -E "s/$match/$replace/g" $file >> $appendTo
    fi
} 
```

Reply:
This function was the one that stood out for me being clearly a candidate for separation as a helper function that's not specific to the doc but just a sed wrapper while the other functions are specific to the specific task of merging the kdocs. I guess I can also separate them between formatting the individual kdocs and merging them.

Reply:
The current separation looks a lot better, it was odd to have only a single function separated from the rest of the scipt.